pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -85,8 +85,7 @@ type ControllerConfigSpec struct {
 	// on all machines.
 	PullSecret *corev1.ObjectReference `json:"pullSecret,omitempty"`
 
-	// images is the map of images that are used by the controller,
-	// i.e. images for etcd Environment Setup, etcd, Kube Client Agent, and Infra containers.
+	// Images is map of images that are used by the controller to render templates under ./templates/
```

Comment:
images at the start of the comment must correspond to the json name, so it has be lowercase here.